x-pack/plugins/observability/public/services/get_news_feed.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { AppMountContext } from 'kibana/public';
+
+export interface NewsItem {
+  title: { en: string };
+  description: { en: string };
+  link_url: { en: string };
+  image_url?: { en: string } | null;
+}
+
+interface NewsFeed {
+  items: NewsItem[] | [];
```

Comment:
```suggestion
  items: NewsItem[];
```

And then you might have to change below
```diff
-return { items: [] };
+return { items: [] as NewsItem[] };
```